csrankings-i.csv

```diff
@@ -98,7 +98,7 @@ Ido Dagan,Bar-Ilan University,http://u.cs.biu.ac.il/~dagan,YzGAGtoAAAAJ
 Ido Tal,Technion,http://webee.technion.ac.il/people/idotal,PgfGXw4AAAAJ
 Idris Skloul Ibrahim,Heriot-Watt University,https://www.hw.ac.uk/staff/uk/macs/idris-ibrahim.htm,NOSCHOLARPAGE
 Ifat Yasin,University College London,http://www.cs.ucl.ac.uk/cs_people/I.Yasin.html,NOSCHOLARPAGE
-Ifeoma Nwogu,Rochester Institute of Technology,https://www.cs.rit.edu/dr-ifoema-nwogu,pCOmTY0AAAAJ
+Ifeoma Nwogu,University at Buffalo,https://cse.buffalo.edu/~inwogu,pCOmTY0AAAAJ

```

Comment:
Home page says currently at RIT.
https://www.cs.rit.edu/~ion/bio.html
https://www.linkedin.com/in/ifeoma-nwogu-7790a34/